### PR TITLE
Add separate build targets for each platform to root Makefile

### PR DESCRIPTION
**Pull request by nicoelayda**
Allows independent building for each platform via either `make 3ds` or `make switch`.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -4,4 +4,10 @@ all:
 	@for dir in $(SUBDIRS); do $(MAKE) -C $$dir; done
 
 clean:
-	@for dir in $(SUBDIRS); do $(MAKE) clean -C $$dir; done
\ No newline at end of file
+	@for dir in $(SUBDIRS); do $(MAKE) clean -C $$dir; done
+
+nds:
```
Comment (BernardoGiordano) [inline Makefile]: Change this to `3ds`

File: Makefile [hunk 2]
```diff
@@ -4,4 +4,10 @@ all:
...
+	$(MAKE) -C 3ds
+
+nx:
```
Comment (BernardoGiordano) [inline Makefile]: Change to `switch`

File: Makefile (hunk 1 above)
Comment (nicoelayda) [reply Makefile]: Fixed

File: Makefile (hunk 2 above)
Comment (nicoelayda) [reply Makefile]: Fixed

**Pull request merged**